capabilities/classification/guide.ipynb

```diff
@@ -43,8 +43,8 @@
    "source": [
     "import os\n",
     "\n",
-    "os.environ['VOYAGE_API_KEY'] = \"VOYAGE KEY HERE\"\n",
-    "os.environ['ANTHROPIC_API_KEY'] = \"ANTHROPIC KEY HERE\""
+    "os.environ[\"VOYAGE_API_KEY\"] = \"VOYAGE KEY HERE\"\n",
```

Comment:
Were these changes through a linter? If so, I think we should codify the linter and have it run on every commit. I might also suggest separating the model changes from linting changes into separate PRs to make it easier to revert if anything breaks.